### PR TITLE
Fix #6675 #6716 #6717 #6688 IE9/IE10/IE11 issues

### LINKED ISSUE #6675: Genomebrowser issues using IE11
### How to Reproduce
- Use **IE11**
- Sign in as superuser
- Extract and import [test.zip](https://github.com/molgenis/molgenis/files/1317919/test.zip) using the 'Advanced data import' plugin
- Navigate to the 'Data explorer' plugin and select 'test' in the dropdown

### Expected behavior
- Genomebrowser displays track information

### Observed behavior
```
Error! An error occurred. Please contact the administrator.
'Symbol' is undefined
```
- Genes, Repeats and Conservation rows in the genome browser display 'Fetching stylesheet'

### LINKED ISSUE #6688: Data explorer filter removal results in IE11 error
### How to Reproduce
- Use **IE11**
- Sign in as superuser
- Import [it_emx_datatypes.xlsx](https://github.com/molgenis/molgenis/files/1320256/it_emx_datatypes.xlsx) using the 'Advanced data import' plugin
- Navigate to the 'Dataexplorer' plugin
- Select 'TypeTest' in the dropdown
- Select 'xint label' in the 'Data item selection' panel and create a filter with value '1'
- Select 'xlong label' in the 'Data item selection' panel and create a filter with value '2'
- Select the 'x' icon next to 'xint label: (1 ≤ x)' in the 'Data item filters' panel
- Select the 'x' icon next to 'xlong label: (2 ≤ x)' in the 'Data item filters' panel

### Expected behavior
- Filters removed successfully, data table displays all rows

### Observed behavior
- Filter is removed from the list, data table is not updated, error is displayed:
```
Error! An error occurred. Please contact the administrator.
SyntaxError: Expected "(" or [^"'();,=!~<> ] but end of input found.
```

### PR DESCRIPTION
**Pull request by dennishendriksen**
#### Checklist
- [x] Functionality works & meets specifications
- [x] Code reviewed
- [x] Code unit/integration/system tested
- [x] User documentation updated
- [x] (If you have changed REST API interface) view-swagger.ftl updated
- [x] Test plan template updated
- [x] Clean commits

### CODE REVIEW

File: molgenis-core-ui/src/main/resources/templates/molgenis-header.ftl [hunk 1]
```diff
@@ -75,13 +76,6 @@
         <script src="https://apis.google.com/js/platform.js<#if authenticated?? && authenticated>?onload=onLoad</#if>"
                 async defer></script>
     </#if>
-
-    <!--[if IE 9]>
-    <#-- used to disable the genomebrowser in IE9 -->
-        <script>top.molgenis.ie9 = true;</script>
-        <#-- required by dalliance-compiled.js to load the genomebrowsers in IE9 -->
-        <script src="<@resource_href "/js/typedarray.min.js"/>  </script>
-    <![endif]-->
```
Comment (Mark-de-Haan) [inline molgenis-core-ui/src/main/resources/templates/molgenis-header.ftl]: Step 1: disable genome browser for IE9
Step 2: load js to be able to load genome browser for IE9

Brilliant stuff 🤣

File: molgenis-core-ui/src/main/resources/templates/molgenis-header.ftl [hunk 2]
```diff
@@ -42,6 +43,10 @@
     <#if app_settings.cssHref?has_content>
         <link rel="stylesheet" href="<@resource_href "/css/${app_settings.cssHref?html}"/>" type="text/css">
     </#if>
+    <#if !version?? || version == 1>
```
Comment (Mark-de-Haan) [inline molgenis-core-ui/src/main/resources/templates/molgenis-header.ftl]: So we are not loading the polyfills for Vue plugins.
The Menu.vue will do it for us?

Comment (dennishendriksen) [reply molgenis-core-ui/src/main/resources/templates/molgenis-header.ftl]: That's up to the version > 1 code that includes the header. Currently polyfills are imported in vue plugins.

File: molgenis-core-ui/src/main/resources/templates/molgenis-header.ftl (hunk 1 above)
Comment (dennishendriksen) [reply molgenis-core-ui/src/main/resources/templates/molgenis-header.ftl]: ;-)

The code was moved to view-dataexplorer-mod-data.ftl without the typedarray script tag

**Comment by fdlk** [comment]
IE9
* Main menu shows again for new style plugins
* Footer shows again for new style plugins
* Data explorer shows data
* for old plugins the degradation is a bit less complete than it used to be, but still easy to find stuff that breaks. E.g. I cannot create a mapping project.
* for new plugins the menu is still broken

IE10
* Main menu shows again
* Footer shows
* Old plugins are usable
* Data Explorer is usable, and #6688 is fixed.
* Genome browser broken when server runs on http://localhost:8080/ ("Resource has already been provided" / "Loading stylesheets")
* Navigator is usable
* Mapping service is usable
* Metadata Editor looks usable

**Comment by dennishendriksen** [comment]
@fdlk Good suggestion, I've added [Dev guide: 5.3.5 Cross-browser compatibility](https://docs.google.com/document/d/1HrEp_u41GZ3rYD_iCIrnXleoyzD_eaPebVwa4cJe5L4/edit#heading=h.soburxlvbe7k) including a comment that I would like to ask you to take a look at.

**Comment by fdlk** [comment]
See also #6734

**Pull request merged**